packages/accounts-password/password_tests.js

```diff
@@ -1697,6 +1697,34 @@ if (Meteor.isServer) (function () {
     }
   )
 
+  Tinytest.add(
+    'passwords - enroll tokens don\'t get cleaned up when reset tokens are cleaned up',
+    function (test) {
+      var email = test.id + '-intercept@example.com';
+      var userId = Accounts.createUser({email: email, password: 'password'});
+      Accounts.sendEnrollmentEmail(userId, email);
+      test.isTrue(!!Meteor.users.findOne(userId).services.password.reset);
```

Comment:
Yes exactly. Maybe it's better to test for equality then directly.
` var resetToken = Meteor.users.findOne(userId).services.password.reset;`
`test.isTrue(resetToken);`
`Accounts._expirePasswordResetTokens(new Date(), userId);`
`test.isEqual(resetToken, Meteor.users.findOne(userId).services.password.reset);`